src/server.cc

```diff
@@ -50,6 +50,18 @@ Server::~Server() {
   }
 }
 
+// Kvrocks threads list:
```

Comment:
@karelrooted all threads, please review

Reply:
Confirmed:  all thread is included in the current thread list

Reply:
@karelrooted thanks